resources/log4j.properties

```diff
@@ -17,13 +17,13 @@ log4j.appender.file.layout.ConversionPattern=%d [%t] %-5p%c - %m%n
 log4j.appender.metabase=metabase.logger.Appender
 
 # customizations to logging by package
-log4j.logger.metabase.driver=DEBUG
-log4j.logger.metabase.middleware=DEBUG
-log4j.logger.metabase.models.permissions=DEBUG
-log4j.logger.metabase.query-processor.permissions=DEBUG
-log4j.logger.metabase.query-processor=DEBUG
-log4j.logger.metabase.sync=DEBUG
-log4j.logger.metabase.models.field-values=DEBUG
+log4j.logger.metabase.driver=INFO
+log4j.logger.metabase.middleware=INFO
+log4j.logger.metabase.models.permissions=INFO
+log4j.logger.metabase.query-processor.permissions=INFO
+log4j.logger.metabase.query-processor=INFO
+log4j.logger.metabase.sync=INFO
+log4j.logger.metabase.models.field-values=INFO
```

Comment:
@salsakran @camsaul All of these entries are not needed as below this we set logger.metabase to INFO. I don't feel strongly about leaving them in or removing them. I left them in so it would be easy for users to tweak the logging levels as needed.

Reply:
FWIW I think it makes sense to keep the sync logging levels at `DEBUG` sfor a release or two since all the sync code was completely rewritten for 0.26. That will make debugging any potential issues that come up easier. But the query processor stuff is pretty stable and runs a thousand times more often than sync so taking that stuff out makes sense